brew.sh

```diff
@@ -3,15 +3,30 @@
 # Install Brew if not installed
 command -v brew > /dev/null || /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-# Update Brew and Formulae
+# Update Brew and Formula
```

Comment:
```suggestion
# Update Brew and Formulae
```

Formulae is the correct spelling of the plural form of formula. See https://formulae.brew.sh/

Reply:
Thank you Harry. I didn't know that "Formulae" is an alternative plural of formula. Anyway It means the same thing as "Formulas" but how on earth I missed "s".

Reply:
Brew uses the more scientific and British-English formulae.

Reply:
By the way this comment "#Update Brew and Formulae" and this action "brew update" means to me something totally different. I wasn't aware what is the purpose of brew formula:

So now I know what is the meaning:
A formula provides instructions and metadata for Homebrew to install a piece of software. Every Homebrew formula is a Formula.

Reply:
Yup, Brew has 2 components - package manager and a registry of instructions.

Reply:
This suggestion still stands,

Reply:
This suggestion still stands.